src/Sulu/Bundle/DocumentManagerBundle/Command/PHPCRCleanupCommand.php

```diff
@@ -192,20 +182,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 }
                 if (0 !== $status) {
                     ++$stats['erroredNodes'];
-                    $erroredUuids[] = $uuid;
+                    $erroredUuids[$uuid] = $process->getErrorOutput();
```

Comment:
@mamazu we should also write this to the debug markdown file! then it also would be possible to have that for lots of errored nodes

Reply:
Error output should now be also logged.